packages/ceramic-doctype-tile/src/tile-doctype.ts

```diff
@@ -78,18 +87,23 @@ export class TileDoctype extends Doctype {
 
         const { content } = params
         const record = { doctype: DOCTYPE, data: content, header: metadata, unique }
-        return TileDoctype._signDagJWS(record, context.did, metadata.owners[0])
+        if (!is3id) {
+            return TileDoctype._signDagJWS(record, context.did, metadata.owners[0])
+        }
```

Comment:
To sign the genesis record or not should be an option passed. 3ID documents should actually have a signed genesis record (in most cases).

Reply:
now every 3ID genesis record is being signed